help/sources/tutorials/ui/create/analytics/pendo-webhook.md

```diff
@@ -0,0 +1,165 @@
+---
+title: Create a Pendo source connection in the UI 
+description: Learn how to create a Pendo source connection using the Adobe Experience Platform UI.
+---
+# (Beta) Create a [!DNL Pendo] source connection in the UI
+
+>[!NOTE]
+>
+>The [!DNL Pendo] source is in beta. See the [sources overview](../../../../home.md#terms-and-conditions) for more information on using beta-labelled sources.
+
+This tutorial provides steps for creating a [!DNL Pendo] source connection using the Adobe Experience Platform user interface.
+
+## Getting started {#getting-started}
+
+This tutorial requires a working understanding of the following components of Experience Platform:
+
+* [[!DNL Experience Data Model (XDM)] System](../../../../../xdm/home.md): The standardized framework by which [!DNL Experience Platform] organizes customer experience data.
+  * [Basics of schema composition](../../../../../xdm/schema/composition.md): Learn about the basic building blocks of XDM schemas, including key principles and best practices in schema composition.
+  * [Schema Editor tutorial](../../../../../xdm/tutorials/create-schema-ui.md): Learn how to create custom schemas using the Schema Editor UI.
+* [[!DNL Real-Time Customer Profile]](../../../../../profile/home.md): Provides a unified, real-time consumer profile based on aggregated data from multiple sources.
+
+## Prerequisites - Sample JSON to define source schema for [!DNL Pendo] {#prerequisites-json-schema}
+
+Before creating a [!DNL Pendo] source connection, you will require a source schema to be provided. You can use the below JSON.
+
+```
+{
+  "accountId": "58f79ee324d3f",
+  "timestamp": 1673372516,
+  "visitorId": "test@test.com",
+  "uniqueId": "166e50cdf40930fe1367e4d44795c9c74d88b83a",
+  "properties": {
+    "guideProperties": {
+  "name": "Guide Conversion Test"
+  }
+}
+}
+```
+
+Refer to the https://support.pendo.io/hc/en-us/articles/360032285012-Webhooks
```

Comment:
"For more information, please refer to the [[!DNL Pendo] guide](https://support.pendo.io/hc/en-us/articles/360032285012-Webhooks) on webhooks.